xsd/netex_framework/netex_responsibility/netex_responsibility_support.xsd

```diff
@@ -268,84 +107,4 @@ Rail transport, Roads and Road transport
 		</xsd:simpleContent>
 	</xsd:complexType>
 	<!-- ======================================================================= -->
-	<xsd:simpleType name="ResponsibilityRoleIdType">
```

Comment:
For reference: this went into ./netex_framework/netex_responsibility/netex_responsibilitySet_support.xsd: